src/main/java/com/raisetech/springbootstudy/controller/NameController.java

```diff
@@ -15,10 +20,21 @@
 
 @RestController
 public class NameController {
+
+  private final NameService nameService;
+
+  public NameController(NameService nameService) {
+    this.nameService = nameService;
+  }
+
   // GET
   @GetMapping("/names")
-  public List<String> getNames() {
-    return List.of("enami", "koyama", "noda", "akamatsu");
+  public List<String> getNames() { // Listで返す形に変更
+    return nameService.findAll()
+        .stream()
+        .map(NameResponse::new) // NameResponseのコンストラクタにNameを渡す
+        .map(NameResponse::getName) // NameResponseのgetNameメソッドを呼び出す
```

Comment:
List<String>で返すなら下記のようにしてもいいと思いました。
```suggestion
        .map(Name::getName)
```

Reply:
深い…！

Reply:
@yoshi-koyama 
これはNameResponseを入れるための妥当な理由がない（整形も要らない）ので、経路を複雑化せずにNameのgetterをそのまま使いなさいという理解で良いでしょうか？？？

Reply:
あってます！
この実装ならNameResponse不要かなと思います。

Reply:
ありがとうございます！
ではこれで進めます！